programs/bpf_loader/src/syscalls/mod.rs

```diff
@@ -1725,7 +1730,11 @@ declare_syscall!(
                     .map_err(SyscallError::InstructionError),
                 result
             );
-            if instruction_context.get_stack_height() == TRANSACTION_LEVEL_STACK_HEIGHT
+            if stop_sibling_instruction_search_at_rising_edge {
```

Comment:
Why is this called a rising edge if we break when the next instruction context is at a lower stack height?

Reply:
Because it is a reverse search. I guess I should rename `prev_stack_height` to `stack_height_of_successor`.

Reply:
Actually, now that I think about it, it should not detect any rising edge but just the one going from `stack_height-1` to `stack_height`. So, we don't need to track the `prev_stack_height` / `stack_height_of_successor` at all.

Reply:
Ah ok, makes sense, I'm not too worried about the rename but a comment explaining what a rising edge is would be nice